q2_dada2/_denoise.py

```diff
@@ -44,6 +44,10 @@ def _check_featureless_table(fp):
     'trim_left_r': _WHOLE_NUM,
     'max_ee': _NAT_NUM,
     'trunc_q': _WHOLE_NUM,
+    'chimera_method': _SKIP,
+    # TEST chimera_method FOR VALIDITY HERE?
+    # Value should be 'pooled', 'consensus' or 'none'
```

Comment:
If you wanted to validate independent of the semantic type, you could add: 
`'chimera_method': lambda x: x in {'pooled', 'consensus', 'none'},`.

But I think ideally this `_valid_inputs` dict would be removed entirely in the future (using `qiime2.plugin.Range`).